### DISCUSSION TITLE
Why Zitadel needs admin access to the database?

### DISCUSSION THREAD

**Discussion by DBLouis** [zitadel/zitadel#7923] [Q&A] [answered]
I'm trying to setup Zitadel and I notice that it requires it's own database access of course but also full admin access to postgres? I'm assuming since there are two sets of parameters database_postgres_user* and database_postgres_admin*. Does Zitadel needs to create additional databases?

---

Side question: Do you support/recommend using a postgres database shared with other applications?

**Comment by hifabienne** [comment]
@adlerhurst @muhlemmer can you help here?

**Comment by DBLouis** [comment, marked as answer]
I think I found my answer here: [link](https://zitadel.com/docs/self-hosting/manage/database#zitadel-with-postgres).

> 
> The admin user is only needed for the installation step but can also be avoided by preparing the necessary requirements:
> 
>   - the zitadel user
>   - the zitadel database
>   - granting required permissions to the zitadel user
